odoo/addons/base/models/ir_model.py

```diff
@@ -1969,7 +1969,10 @@ def _compute_complete_name(self):
     @api.depends('model', 'res_id')
     def _compute_reference(self):
         for res in self:
-            res.reference = "%s,%s" % (res.model, res.res_id)
+            reference = ''
+            if res.model and res.id != 0:
```

Comment:
Why using `res.id` instead of of `res.res_id` ? (maybe you can rename `res` in `rec` to avoid the confusion)

Also `res.res_id != 0` can be transform into `res.res_id`. `0` is Falsy anyway.

Reply:
@ryv-odoo updated the pr as suggested